express/routes/seasons.js

```diff
@@ -0,0 +1,56 @@
+const express = require('express');
+const httpStatus = require('http-status');
+const { validate } = require('express-validation');
+
+const seasonsService = require('../services/seasons');
+const validations = require('../validations/seasons')
+
+const router = express.Router();
+
+router.get('/', async (req, res) => {
+	const seasons = await seasonsService.getAll();
+
+	return res.send({ seasons });
+});
+
+router.get('/:id', validate(validations.get), async (req, res) => {
+	const season = await seasonsService.getById(req.params.id);
+
+	if (!season) {
+		return res.sendStatus(httpStatus.NOT_FOUND);
+	}
+
+	return res.send({ season });
+});
+
+router.post('/', validate(validations.post), async (req, res) => {
```

Comment:
might be it's a bit better to keep '/' post next to the '/' get

Reply:
Vanya, can you explain what did you mean?

Reply:
I mean move ``` router.post('/', validate(validations.post), async (req, res) => { ``` to the line 15